code/check.py

```diff
@@ -1,32 +1,36 @@
+#!/usr/bin/env python
 import sys
 import numpy
 
 
 def main():
-    script = sys.argv[0]
     filenames = sys.argv[1:]
-    if len(filenames) <= 1:  # nothing to check
-        print('Only 1 file specified on input')
+    total_files = len(filenames)
+    if total_files <= 1:  # nothing to check
+        print(f'''{'Only one file' if total_files else 'None files'} specified
+        on input''')
     else:
         nrow0, ncol0 = row_col_count(filenames[0])
-        print('First file %s: %d rows and %d columns' % (
-            filenames[0], nrow0, ncol0))
+        print(f'''First file {filenames[0]}: {nrow0} rows and {ncol0}
+        columns''')
         for filename in filenames[1:]:
             nrow, ncol = row_col_count(filename)
             if nrow != nrow0 or ncol != ncol0:
-                print('File %s does not check: %d rows and %d columns'
-                      % (filename, nrow, ncol))
+                print(f'''File {filename} does not check: {nrow} rows
+                and {ncol} columns''')
             else:
-                print('File %s checks' % filename)
-        return
+                print(f'File {filename} checks')
+
+    return None
 
 
 def row_col_count(filename):
     try:
-        nrow, ncol = numpy.loadtxt(filename, delimiter=',').shape
-    except ValueError:
-        # This occurs if the file doesn't have same number of rows and columns,
-        # or if it has non-numeric content
```

Comment:
I am wondering if you have tried to run this code with your change (I haven't). The default value for the [`delimiter` parameter](https://numpy.org/doc/stable/reference/generated/numpy.loadtxt.html) is whitespace, so when trying to load our inflammation datasets (which is specified in the challenge this code aligns with) without setting `delimiter=','`, our data isn't parsed correctly. I think we would get a `ValueError` because of the commas, and the outcome of the code would not be as expected. For this reason, I don't think we can remove the `delimiter` parameter from our `loadtxt` calls.

Reply:
My bad, I need to check it again since I use self-made files for testing.